packages/kit/src/core/config/options.js

```diff
@@ -171,8 +171,6 @@ const options = object(
 				files: fun((filename) => !/\.DS_STORE/.test(filename))
 			}),
 
-			ssr: boolean(true),
```

Comment:
maybe add a note to tell people it's part of `resolve` now

Reply:
done